src/TileMap/index.js

```diff
@@ -0,0 +1,344 @@
+// import TileMap from 'cfpb-chart-builder/src/js/charts/TileMap';
+
+import accessibility from 'highcharts/modules/accessibility';
+import Highcharts from 'highcharts/highmaps';
+import { STATE_TILES } from './constants'
+
+/* ----------------------------------------------------------------------------
+   Utility Functions */
+/**
+* A reducer function to process the maximum value in the state complaint data
+*
+* @param {number} accum the current max value
+* @param {Object} stateComplaint a candidate value
+* @returns {string} the maximum between the current and a state entry
+*/
+export function findMaxComplaints( accum, stateComplaint ) {
+  return Math.max( accum, stateComplaint.displayValue );
+}
+
+/**
+ * helper function to get the bins for legend and colors, etc.
+ * @param {Array} data all of the states w/ displayValue, complaintCount, raw
+ * @param {Array} colors an array of colors
+ * @returns {Array} the bins with bounds, name, and color
+ */
+export function getBins( data, colors ) {
+  const binCount = colors.length;
+  const max = data.reduce( findMaxComplaints, 0 );
+  const min = 1;
+
+  // Early exit
+  if ( max === 0 ) return [];
+
+  const step = ( max - min + 1 ) / binCount;
+  const bins = [ { from: 0, to: min, color: '#fff', name: 'N/A' } ];
+
+  for ( let i = 0, curr = min; i < binCount; i++, curr += step ) {
+    const minValue = Math.round( curr );
+    const displayValue = Math.round( curr ).toLocaleString();
+
+    bins.push( {
+      from: minValue,
+      to: Math.round( curr + step ),
+      color: colors[i],
+      name: `≥ ${ displayValue }`
+    } );
+  }
+
+  // The last bin is unbounded
+  // eslint-disable-next-line no-undefined
+  bins[bins.length - 1].to = undefined;
+
+  return bins;
+}
+
+/**
+ * helper function to get the per Capita bins for legend and colors, etc.
+ * @param {Array} data all of the states w/ displayValue, complaintCount, raw
+ * @param {Array} colors an array of colors
+ * @returns {Array} contains bins with bounds, colors, name, and color
+ */
+export function getPerCapitaBins( data, colors ) {
+  const binCount = colors.length;
+  const max = data.reduce( findMaxComplaints, 0 );
+  const min = 0;
+
+  // Early exit
+  if ( max === 0 ) return [];
+
+  const step = ( max - min ) / binCount;
+  const bins = [];
+
+  for ( let i = 0, curr = min; i < binCount; i++, curr += step ) {
+    const minValue =  parseFloat( curr.toFixed( 2 ) )
```

Comment:
getting an eslint nag here about multiple spaces

Reply:
This is what happens when I only run `npm test` vs `npm start`